lms/djangoapps/courseware/access.py

```diff
@@ -269,6 +270,9 @@ def _can_enroll_courselike(user, courselike):
     if _has_staff_access_to_descriptor(user, courselike, course_key):
         return ACCESS_GRANTED
 
+    if _is_ccx_course(courselike):
+        return ACCESS_DENIED
```

Comment:
Won't this block legitimate enrollment of students into CCX courses?

Reply:
I'm pretty sure we determined that this doesn't, because students should only be able to enroll if an instructor enrolls them. I'm not aware of other possible enrollment paths valid for ccx courses?